### PR TITLE
python3Packages.lima: init at 0.5

### PR DESCRIPTION
**Pull request by zhaofengli**
###### Motivation for this change

Lima is a Python library that converts Python objects to native data structures.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/lima/default.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi, pytest }:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/lima/default.nix:2]:
```suggestion
{ lib, buildPythonPackage, fetchPypi, pytestCheckHook }:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/lima/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  pname = "lima";
+  version = "0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qqj0053r77ppkcyyk2fhpaxjzsl1h98nf9clpny6cs66sdl241v";
+  };
+
+  checkInputs = [ pytest ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/lima/default.nix:11]:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

**Comment by SuperSandro2000** [comment]
You messed something up while rebasing. Please drop the extra commits.

**Comment by zhaofengli** [comment]
By the way, I'm planning to add a few more Python dependencies for [powerdns-admin](https://github.com/ngoduykhanh/PowerDNS-Admin). Should I create separate PRs for each of them including powerdns-admin itself, or can I batch them into one PR?

**Comment by SuperSandro2000** [comment]
> Should I create separate PRs for each of them including powerdns-admin itself, or can I batch them into one PR?

If they are all required for powerdns-admin and are pretty simple than you can put them into the PR of powerdns-admin. If there are more complicated ones then the merge could be a bit delayed because we need to review all of them. We can always split something out if the PR grows to big.

**Comment by zhaofengli** [comment]
Opening a new PR for powerdns-admin along with all missing dependencies.

**Pull request closed without merging**